mon-pix/app/components/challenge-item-generic.js

```diff
@@ -23,6 +26,11 @@ export default class ChallengeItemGeneric extends Component {
     }
   }
 
+  willDestroy() {
+    super.willDestroy(...arguments);
+    this._removeEventListeners();
+  }
```

Comment:
**question:** Est-ce qu'on est sûr que la ligne `this._removeEventListeners()` sera bien appelée même après l'appel du super ? 
```suggestion
  willDestroy() {
    this._removeEventListeners();
    super.willDestroy(...arguments);
  }
```